addons/account_sequence/models/account_journal.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import fields, models
+
+
+class AccountJournal(models.Model):
+    _inherit = 'account.journal'
+
+    sequence_id = fields.Many2one('ir.sequence', string='Entry Sequence',
+                                  help='This field contains the information related to the numbering of the journal entries of this journal.',
+                                  copy=False, check_company=True)
+    refund_sequence_id = fields.Many2one('ir.sequence', string='Credit Note Entry Sequence',
+                                         help='This field contains the information related to the numbering of the credit note entries of this journal.',
+                                         copy=False, check_company=True)
```

Comment:
> This field contains...

A bit too technical for a tooltip

________________
Regarding the spacing, just a personal preference:
```suggestion
    sequence_id = fields.Many2one(
        comodel_name='ir.sequence', 
        string='Entry Sequence',
        copy=False, 
        check_company=True,
        help="This field contains the information related to the numbering of the journal entries of this journal.",
    )
    refund_sequence_id = fields.Many2one(
        comodel_name='ir.sequence', 
        string='Credit Note Entry Sequence',
        copy=False, 
        check_company=True,
        help="This field contains the information related to the numbering of the credit note entries of this journal.",
    )
```